services/espnet-tts-fr/src/app.py

```diff
@@ -40,6 +41,30 @@
 app = Flask(__name__)
 
 
+def isfloat(num):
+    try:
+        float(num)
+        return True
+    except ValueError:
+        return False
```

Comment:
I added this because there's (supposedly) no built-in function in Python to check if a number is a float.